Tasks/DeployVisualStudioTestAgent/task.json

```diff
@@ -44,14 +44,26 @@
             "required": true,
             "helpMarkDown": "Name of the Test Machine Group on which tests need to be run.",
             "groupName": "testMachineGroups"
-        },       
+        },
+        {
+            "name":"ResourceFilteringMethod",
```

Comment:
test compat scenario
1) existing build definition queue the build again after updating the task
2) task is new and agent is old.